packages/core/src/routes/session/password.test.ts

```diff
@@ -116,60 +99,52 @@ describe('session -> password routes', () => {
     ],
   });
 
-  describe('POST /session/sign-in/password/username', () => {
-    it('assign result and redirect', async () => {
-      interactionDetails.mockResolvedValueOnce({ params: {} });
-      const response = await sessionRequest.post(`${signInRoute}/username`).send({
-        username: 'username',
-        password: 'password',
-      });
-      expect(response.statusCode).toEqual(200);
-      expect(response.body).toHaveProperty('redirectTo');
-      expect(interactionResult).toHaveBeenCalledWith(
-        expect.anything(),
-        expect.anything(),
-        expect.objectContaining({ login: { accountId: 'user1' } }),
-        expect.anything()
-      );
-    });
-
-    it('throw if user not found', async () => {
-      interactionDetails.mockResolvedValueOnce({ params: {} });
-      const response = await sessionRequest.post(`${signInRoute}/username`).send({
-        username: 'notexistuser',
-        password: 'password',
-      });
-      expect(response.statusCode).toEqual(400);
+  it('POST /session/sign-in/password/username', async () => {
+    interactionDetails.mockResolvedValueOnce({ params: {} });
+    const response = await sessionRequest.post(`${signInRoute}/username`).send({
+      username: 'username',
+      password: 'password',
     });
+    expect(response.statusCode).toEqual(200);
+    expect(response.body).toHaveProperty('redirectTo');
+    expect(interactionResult).toHaveBeenCalledWith(
+      expect.anything(),
+      expect.anything(),
+      expect.objectContaining({ login: { accountId: 'id' } }),
+      expect.anything()
+    );
+  });
 
-    it('throw if user found but wrong password', async () => {
-      interactionDetails.mockResolvedValueOnce({ params: {} });
-      const response = await sessionRequest.post(`${signInRoute}/username`).send({
-        username: 'username',
-        password: '_password',
-      });
-      expect(response.statusCode).toEqual(400);
+  it('POST /session/sign-in/password/email', async () => {
```

Comment:
nit: do we also need to check the case of inputting wrong password?

Reply:
It is already covered in line 142